frontend/src/components/pages/CardDetails/Presentation.jsx

```diff
@@ -87,59 +120,65 @@ function CardBasicDetails({ card }) {
           {/* <StoryPoints storyPoints={card.storyPoints} /> */}
           <CardStatusChip card={card} />
         </Grid>
-        <Grid item xs={12} sm={12} md={12}>
-          <Paper className={classes.sectionPaper} variant="outlined">
-            <Typography variant="subtitle2">Assignees:</Typography>
-            <AssigneesList
-              userNames={card.assigneeNames}
-              userIds={card.assignees}
-            />
-            <Typography variant="subtitle2">Reviewers:</Typography>
-            <ReviewersTable
-              reviewerUserEmails={card.reviewerNames}
-              reviewerUsers={card.reviewers}
-              usersThatReviewedSinceLastReviewRequestEmails={
-                card.usersThatReviewedSinceLastReviewRequestEmails
-              }
-              usersThatReviewedSinceLastReviewRequest={
-                card.usersThatReviewedSinceLastReviewRequest
-              }
-            />
-          </Paper>
-        </Grid>
-        <Grid item xs={12} sm={12} md={12}>
-          <Paper className={classes.sectionPaper} variant="outlined">
-            <Table>
-              <TableBody>
-                <TableRow>
-                  <TableCell>Due Time</TableCell>
-                  <TableCell>{dueTime}</TableCell>
-                </TableRow>
-                {card.startTime && (
+        <Grid container spacing={3}>
```

Comment:
I stand to be corrected, but wont this cause a warning? 

AFAIK you can't upt a container directly inside another container unless it's an item.

Reply:
It seems to be in line with the material UI documentation. https://v4.mui.com/components/grid/